contracts/clients/src/BlsWalletWrapper.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable camelcase */
```

Comment:
nit - If we disable this rule for the file we could remove the disables on the individual lines

Reply:
https://github.com/web3well/bls-wallet/pull/390